tools/bazel-tools-tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
sadness, why another tsconfig file? the editor won't know about these settings.

Reply:
The default bazel one doesn't have `types: node` and `lib: es2015`. I removed the other two options.